include/deal.II/lac/arpack_solver.h

```diff
@@ -343,6 +357,52 @@ class ArpackSolver : public Subscriptor
         std::vector<VectorType> &          eigenvectors,
         const unsigned int                 n_eigenvalues = 0);
 
+  /**
+   * Solve the eigensprectrum problem $A x=\lambda x$ by calling the
+   * <code>dsaupd</code> and <code>dseupd</code> or <code>dnaupd</code> and
+   * <code>dneupd</code> functions of ARPACK.
+   *
+   * The function returns a vector of eigenvalues of length <i>n</i> and a
+   * vector of eigenvectors of length <i>n</i> in the symmetric case
+   * and of length <i>n+1</i> in the non-symmetric case. In the symmetric case
+   * all eigenvectors are real. In the non-symmetric case complex eigenvalues
+   * always occur as complex conjugate pairs. Therefore the eigenvector for an
+   * eigenvalue with nonzero complex part is stored by putting the real and
+   * the imaginary parts in consecutive real-valued vectors. The eigenvector
+   * of the complex conjugate eigenvalue does not need to be stored, since it
+   * is just the complex conjugate of the stored eigenvector. Thus, if the last
+   * n-th eigenvalue has a nonzero imaginary part, Arpack needs in total n+1
+   * real-valued vectors to store real and imaginary parts of the eigenvectors.
+   *
+   * @param A The operator for which we want to compute eigenvalues.
+   *
+   * @param eigenvalues is a vector of complex numbers in which the
```

Comment:
Given how this is rendered, say "@param[out] eigenvalues A vector of complex..."